dapp_manager/dapp_manager.py

```diff
@@ -7,18 +7,20 @@
 from datetime import datetime, timedelta
 from pathlib import Path
 from time import sleep
-from typing import List, Union
+from typing import Iterator, List, Union
 
 import appdirs
 import psutil
 
 from .dapp_starter import DappStarter
 from .exceptions import AppNotRunning
-from .storage import RunnerFileType, SimpleStorage
+from .storage import RunnerReadFileType, SimpleStorage
 
 PathType = Union[str, os.PathLike]
 
-COMMAND_OUTPUT_INTERVAL = 1.0
+COMMAND_OUTPUT_INTERVAL = timedelta(seconds=1)
```

Comment:
hmm... again, I see no strong benefit from forcing this value to be of `timedelta` type... to me it's obvious that - if it's a scalar float, it's the number of seconds... and we never use it in a way that would make us benefit from it being a `timedelta` (that would be the case if we wanted to perform comparisons between `datetime`s, etc)

Reply:
As discussed offline, using `timedelta` looks more friendly than sole value.